checks.d/mongo.py

```diff
@@ -875,3 +887,49 @@ def total_seconds(td):
 
         else:
             self.log.debug('"local" database not in dbnames. Not collecting ReplicationInfo metrics')
+
+        # get collection level stats
+        try:
+            # Ensure that you're on the right db
+            db = cli[db_name]
+            # grab the collections from the database
+            coll_names = db.collection_names()
+            collection_metrics = []
+            for m in self.COLLECTION_METRICS:
+                # grab all the collections
+                collection_metrics.append(m.split('.')[1:])
+
+            # loop through the collections
+            for coll_name in coll_names:
+                # grab the stats from the collection
+                stats = db.command("collstats", coll_name)
+                # loop through the metrics
+                for m in collection_metrics:
+                    coll_tags = tags + ["db:%s" % db_name, "collection:%s" % coll_name]
+                    value = None
+                    try:
+                        # descend into the dict to grab the value we're looking for
+                        for c in m:
```

Comment:
Looking at the the content of COLLECTION_METRICS, collection_metrics will be a list of lists with a single element each.

So there will be just one element in m